src/olympia/addons/models.py

```diff
@@ -1509,6 +1509,34 @@ def watch_disabled(old_attr=None, new_attr=None, instance=None, sender=None,
             file_.hide_disabled_file()
 
 
+@Addon.on_change
+def watch_changes(old_attr=None, new_attr=None, instance=None, sender=None,
+                  **kwargs):
+    if old_attr is None:
+        old_attr = {}
+    if new_attr is None:
+        new_attr = {}
+
+    changes = {
+        x for x in new_attr
+        if not x.startswith('_') and new_attr[x] != old_attr.get(x)
+    }
+    basket_relevant_changes = (
+        # Some changes are not tracked here:
+        # - Any authors changes (separate model)
+        # - Creation/Deletion of unlisted version (separate model)
+        # - Name change (separate model, not implemented yet)
```

Comment:
Separate model for name change?  Doesn't changing `name` update `last_updated` on `Addon`?

Reply:
My idea was to listen to translation changes (hence the remark about it being a separate model). Relying on `last_updated` is tricky, because it's modified by a cron that acts on a queryset, making it harder to catch and sync individual changes without completely changing the way it's set.

Reply:
(In any case I made a separate issue for that problem: https://github.com/mozilla/addons-server/issues/12146 )